### ISSUE TITLE
[🕹️] Showcase Status Page somewhere

### ISSUE THREAD

**Issue by Flo4604** [unkeyed/unkey#2185]
### Preliminary Checks

- [X] I have reviewed the documentation: https://unkey.com/docs

- [X] I have searched for existing issues: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Issue Summary

There is https://status.unkey.com, which is cool to have, but it's not showcased anywhere on the site. :/

---

Something for the core team to look into is that the status page is not indexed (when you search unkey status, nothing comes up); if that's by design, then please ignore this comment. 

### Steps to Reproduce

- 

### Expected behavior

I think it would be cool if this were showcased in the navigation, maybe even as its own tab, with a small icon depending on the actual live status (a green icon for everything is fine ) and an orange/red icon for partial/complete outage. 

Maybe incident.io has badges for that.

### Version info

```markdown
-
```

**Comment by chronark** [comment]
Indeed, maybe a widget in the footer would be cool too, let's check out if incident.io has anything like that

/award 150 points

**Comment by Flo4604** [comment]
I didn't find anything on the incident.io page that looks like a widget.
So maybe just a link to it is sufficient? 

Otherwise they have an API so writing a widget could work.
